src/plugins/event_annotation/public/event_annotation_service/helpers.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { euiLightVars } from '@kbn/ui-theme';
+export const defaultAnnotationColor = euiLightVars.euiColorAccent;
```

Comment:
nit: I probably recommended not to fix the value at the time of import. Maybe `geDefaultAnnotationColor = () => euiLightVars.euiColorAccent;`

Reply:
Why does it matter? 🤔

Reply:
@mbondyra not related to alex suggestion, but to the color: the `euiColorAccent` differs if you are in dark mode or light mode. in the light theme the color is  #F04E98 but is  #F68FBE in dark mode.
@MichaelMarcialis ideas?

Reply:
@markov00 does it matter? Why is it concerning?

Reply:
On switching to dark mode the annotations will still have the light theme color because it's putting the hex value into the saved object. I think that is OK for this PR, it's the same for all of the place where we allow the users to pick a color (series, reference lines and so on). When we discussed this the last time we said we would tackle this problem separately in a general way and apply the solutions to all the places which allow this kind of thing right now.

Reply:
> @mbondyra not related to alex suggestion, but to the color: the `euiColorAccent` differs if you are in dark mode or light mode. in the light theme the color is `#F04E98` but is `#F68FBE` in dark mode.
> @MichaelMarcialis ideas?

@markov00: That's a good point; something we hadn't considered in our discussion yesterday. Ideally, we would have a single color value that works well for both light and dark modes. Otherwise, the default color is not a purely static value, but a partially dynamic one that changes depending on the theme mode selected when the visualization was created. This could be unexpected or undesirable to users (especially after changing an existing space's theme mode, when sharing/copying visualization across spaces, and in the future when users can set their own theme modes). This is part of the reason why we chose a gray color for the default reference line color, because it works in both theme modes.

In this case, the difference in the color value between light and dark mode presents a similar problem to the one we discussed yesterday if we were to choose black (which would require switching to white in dark mode). That said, because the contrast gap between accent colors in light and dark mode is far lower than the contrast gap between black and white, perhaps we can live with it for now until a more general solution for handling color selection between light/dark themes is addressed, as @flash1293 stated above.

For the sake of the immediate present, I'd say let's either stick with the EUI accent color (regardless of the fact that it will change on visualization creation depending on theme mode), or switch to same gray we use for reference lines as the default.